### PR TITLE
Fix discussion comment time

### PR DESCRIPTION
**Pull request by ymh6315431**
**What this PR does**:
| before | after |
| ------ | ------ |
|   image |  image    |
|    image  |    image    |

**Which issue(s) this PR fixes**:

Fixes #

**Type of changes**
Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

**Feature/Issue validation/testing**:

Please describe the tests that you ran to verify your changes and relevant result summary. 

- [ ] Test A
- [ ] Test B

- Logs

**Special notes for your reviewer**:

**Checklist**:

- [ ] I have added unit/e2e tests that prove your fix is effective or that this feature works.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] I have reviewed my own code and ensured that it follows the project's style guidelines.

**Release note**:

```release-note

```

### CODE REVIEW

**Comment by esperyong** [comment]
@codegpt /review

**Comment by esperyong** [comment]
@codegpt /review

File: frontend/src/components/community/DiscussionDetails.vue [hunk 1]
```diff
@@ -92,6 +93,10 @@ export default {
     this.getComment(this.discussionId)
   },
   methods: {
+    formatTime(timestamp) {
+      const parsedTime = beiJingTimeParser(timestamp);
+      return format(parsedTime, this.$i18n.locale === 'en' ? 'en_US' : 'zh_CN');
```
[the remark this answers is not in the document]
Comment (hiveer) [reply frontend/src/components/community/DiscussionDetails.vue:98]: @ymh6315431 the suggestion from starship is right, we should add catch here

File: frontend/src/components/community/DiscussionCards.vue [hunk 2]
```diff
@@ -59,12 +60,15 @@
     },
     mounted() {},
     methods: {
+      formatDate (date) {
+        return dayjs(date).format('YYYY-MM-DD HH:mm:ss')
```
[the remark this answers is not in the document]
Comment (hiveer) [reply frontend/src/components/community/DiscussionCards.vue:64]: @ymh6315431 the suggestion from starship is right, we should consider the invalid case

[the remark this answers is not in the document]
Comment (ymh6315431) [reply frontend/src/components/community/DiscussionCards.vue]: done

File: frontend/src/components/community/DiscussionDetails.vue (hunk 1 above)
[the remark this answers is not in the document]
Comment (ymh6315431) [reply frontend/src/components/community/DiscussionDetails.vue]: done

**Review by hiveer — APPROVED** [review]

**Pull request merged**